src/com/untamedears/realisticbiomes/GrowthConfig.java

```diff
@@ -261,103 +238,65 @@ public boolean isPersistent() {
 	}
 	
 	// given a block (a location), find the growth rate using these rules
+	
 	public double getRate(Block block) {
 		if (type == Type.FISHING_DROP) {
 			return getBaseRate(block);
 		}
-		// rate = baseRate * sunlightLevel * biome * (1.0 + soilBonus)
-		double rate = baseRate;
-		// if persistent, the growth rate is measured in growth/second
+		double rate= getBaseRate(block);
+		if (rate==0D) {
+			return rate;
+		}
 		if (isPersistent) {
 			rate = 1.0 / (persistentRate * SEC_PER_HOUR);
+			Double biomeMultiplier = biomeMultipliers.get(block.getBiome());
+			rate *= biomeMultiplier.floatValue();

```

Comment:
Is getBaseRate used by anything else? If not, we could probably do that, the functionality is identical and it's a lot easier to understand.